scrapers.js

```diff
@@ -2048,6 +2048,91 @@ let scrapers = [
+      dataIds.forEach((label) => {
+        if (typeof label['uid'] == typeof 1)
+          labels[label['uid']] = ufs[label['name']]
+      })
+  
+      let data = await fetch.page(this.url, false);
+      let data_json = JSON.parse(data.text().replace('var database=', ''))
+  
+      let data_brazil = []
+
+      data_json['brazil'].filter((row) => row['date'] == [
+        datetime.getDate().toString().padStart(2,0),
+        (datetime.getMonth() +1).toString().padStart(2,0),
+        datetime.getFullYear().toString().padStart(2,0)
+      ].join('/')).forEach((date_data) => {
+        let date = date_data['date'].split('/')
+        let time = date_data['time'].split(':')
+        let data_date = new Date(parseInt(date[2]), parseInt(date[1]) - 1, parseInt(date[0])) 
+        date_data['values'].forEach((value) => {
+          data_brazil.push({
+            date: data_date,
+            country: 'BRA',
+            city: undefined,
+            county: undefined,
+            state: labels[parseInt(value['uid'])][0],
+            country: 'Brazil',
+            cases: value['cases'] || 0,
+            deaths: value['deaths'] || 0,
+            recovered: undefined,
+            tested: value['suspects'] || 0,
```

Comment:
As I could research, suspects is more like presumptive cases. But here they I divided in two levels.
suspeito (suspected) - less possible but not discarted, 
provável (probable) - really probably
from today:
8.819 (80,17%) - suspects
291 (2,65%) - confirmed

Beside the fact that it's part of the protocol, it will hide the current and real cases.
http://coronavirus.saude.gov.br/sobre-a-doenca#casossuspeito

What do you think about remove tested, and maintain cases just with real cases. Because I can't now confirm the methodology utilized to categorize a real suspect.

Reply:
I did a commit to remove the value, because IMO the number presented by the protocol will hide the current situation in the country. There are a lot of fake news, and a big number diverging could put less credibility.